screens/CampaignScreen/Content.js

```diff
@@ -1,52 +1,29 @@
 import React from 'react';
-import { Dimensions, ScrollView } from 'react-native';
+import { Box } from "native-base";
+import { Dimensions, Platform } from 'react-native';
 
-import {
-    Box,
-    VStack,
-    Text,
-    Icon,
-    Progress
-} from "native-base";
+import ContentHeader from './contentheader';
+import Member from './Member';
+import Time from './Time';
+import Paragraph from './Paragraph';
 
-import { CONTENT_DATA } from "./dummyData/content";
-
-const SCREEN_WIDTH = Dimensions.get("window").width;
+const DEVICE_HEIGHT = Dimensions.get('screen').height
 
 export default function Content() {
     return (
-        <ScrollView>
-            <Box >
-                <Box w={'100%'} justifyContent="space-between" flexDirection={'row'} flexWrap={'wrap'} >
-                    {
-                        CONTENT_DATA.map((item, idx) => (
-                            <Box key={idx} bg="bg.600" w={(SCREEN_WIDTH / 2) - 20} h={240} borderRadius={15} color={"white"} mb={10} p={3}>
-                                <VStack>
-                                    <Box bg={"bg.300"} w={10} mt={2} h={10} justifyContent={"center"} alignItems={'center'}
-                                        borderRadius={5} mb={4} >
-                                        <Icon type={item.iconType}
-                                            name={item.iconName}
-                                            color={item.iconColor}
-                                            size={6}
-                                        />
-                                    </Box>
-                                    <Box color={"white"} fontWeight={"bold"} fontSize={18} mb={10} >
-                                        {item.des}
-                                    </Box>
-                                    <Text color={'white'} fontWeight={700} mb={2}>{item.indicatorValue}%</Text>
-                                    <Progress
-                                        size="lg"
-                                        colorScheme="#39357A"
-                                        mb={4}
-                                        value={item.indicatorValue}
-                                        rounded="50" />
-
-                                </VStack>
-                            </Box>
-                        ))
-                    }
-                </Box>
-            </Box>
-        </ScrollView >
+        <Box
+            border={'#333166'}
+            bg={'bg.500'}
+            minHeight={Platform.OS === 'ios' ? DEVICE_HEIGHT - 300 : DEVICE_HEIGHT - 230}
```

Comment:
Why do we need DEVICE_HEIGHT?

Reply:
I am using DEVICE_HEIGHT to give the min-height to middle content section, Without height, flex is not working as expected.